src/commands/sbom/__tests__/valiation.test.ts

```diff
@@ -0,0 +1,16 @@
+import {getValidator, validateSbomFile} from '../validation'
+
+const validator = getValidator()
+
+describe('validation of sbom file', () => {
+  test('should succeed when called on a valid SBOM file', () => {
+    expect(validateSbomFile('./src/commands/sbom/__tests__/fixtures/bom.1.4.ok.json', validator)).toBeTruthy()
+  })
+  test('should fail on files that do not exists', () => {
```

Comment:
```suggestion
  test('should fail on files that do not exist', () => {
```

Reply:
@juli1 also the file name has a typo inside it (`valiation`)